.circleci/config.yml

```diff
@@ -119,7 +136,10 @@ jobs:
       - setup_dependencies
       - run:
           name: "Run Tox"
-          command:  tox
+          no_output_timeout: 50m
```

Comment:
Looks like tests were sometimes failing right on this line. Can that be bumped up to 60m?